Merge another project

Cliwo opened the pull request:
[변경되는 주요 내용들은 다음과 같습니다.]
1. 이제 두 개의 유니티 없이 하나의 유니티 프로젝트로 실험을 가능하게 합니다.
2. Screen Shot을 찍는 것과, excel 파일을 수동으로 옮겨주는 것을 해결합니다. 이제 Assets 폴더나 Desktop폴더로 자동으로 생성시킬 수 있습니다.
3. 입체 실험에서 오브젝트들이 생성되는 위치를 조정하는 스크립트를 추가 했습니다.

[우려가능한 문제점]
2 -> Screen Shot을 찍지 않았을 때, 유니티가 종료되기 직전 스크린샷을 찍는 기능을 넣었으나 정상적으로 동작하지 않습니다. (다 종료되고나서 촬영합니다) 수정이 필요해 보입니다.
3 -> 입체 실험에서 오브젝트들이 생성될 때 directional right 방향이 수정되지 않아, 사용자가 보는 방향이 어둡게 나올 수 있습니다. 회전을 하면서 물체에 입사하는 빛의 량이 쉽게 달라지지만, 혹시 사용해보시고 문제라고 생각되신다면 고쳐야할 것 같습니다.

[각 기능들에 대한 자세한 설명]
1. Strabismus.unity 씬으로 시작합니다. Hierarchy 의 AppManager -> SceneLoader -> Translate Time 값에 따라 자동으로  씬이 전환됩니다. 권장되는 수치는 40초입니다. (현재는 5초)
1-1. AppManager -> LoggingManager 의 Patient ID가 없을 경우 에러가 발생합니다. 유니티 console창의 'Error Pause'를 활성화 시킬 경우 data를 놓치는 경우 없이 실험을 진행 할 수 있습니다. Patient ID로 자동으로 데이터를 분류, 생성할 수 있습니다.

2. AppManager -> LoggingManager로, 기존의 data를 저장하는 방식을 그대로 사용하되, Desktop 또는 Assets 폴더에 Patient ID에 맞는 폴더에 해당 data들을 위치 시킵니다.
2-1. ScreenShotManager로 ScreenShot을 찍을 수 있습니다. 위의 data (csv파일들)이 저장되는 곳과 같은 곳에 저장됩니다. 찍는 버튼은 아래의 그림을 참조하시면 됩니다.
![1](https://user-images.githubusercontent.com/15336467/42270406-bd6937a4-7fbb-11e8-8293-d79f9c6b4d40.png)

3. 이제 간헐적 외사시를 data 수집 Scene 끝나고, 피험자가 어디를 보고 있든, 보고 있는 위치 그 앞에 입체시 검사 object들이 생성됩니다. 
(Feedback이 필요합니다)

아직 모든 코드는 refactoring 전이라 코드가 심히 더럽습니다.
빠른 시일내에 Refactoring 하겠습니다

Cliwo committed:
seperate scene

Cliwo committed:
just organize assets

Cliwo committed:
Merged another project into this project

Cliwo committed:
organize some assets

Cliwo committed:
Edit EyeCamera Manually

Cliwo committed:
In progress 'Polishing Data File Process'

Cliwo committed:
Add Log Manager

Cliwo committed:
temporary saving

Cliwo committed:
1. Change Shader (Standard -> Unlit)
2. Problem on position and rotation when chage scene

Cliwo committed:
delete unused scripts

Cliwo committed:
1. rollback stereopsis materials (use light)
2. fix positioning problem on both scenes
3. Add minor debugging script (draw orientation of HMD)
4. small refactoring (but needs more than this)

Cliwo committed:
1. Disable Debug code
2. Delete Custom Editor for logging Manager

Cliwo committed:
Apply New Logging System
-> Now We can control data location immediatly on Unity Editor while game is running (See AppManager - Logging Manager & ScreenShootManager)
-> You can select Assets folder or Desktop folder with Bool Value (see Logging Manager)

** I Added Fallback for when we forget to save screenshot. but it is not working properly (it captures the screen after application is closed)
We can fix this problem with just capturing screen frequently while game is running but we should be careful of possibility of fps drop

If there is a better solution, pls let me know :) (I didn't apply my idea yet)

Cliwo committed:
Clean csv files

jafffy committed:
Remove vs and vscode setting files

jafffy commented:
확인해 봤는데, 아래와 같은 이슈가 있어서 같이 봐야 할 것 같아요!

- Strabismus scene에서 missing prefab이 발생합니다.
![missing-prefab](https://user-images.githubusercontent.com/5888496/42298822-db41a8d8-8042-11e8-83a9-84bfc3696d46.png)

- 스샷 찍는 기능이 작동을 안한다고 하셨는데, 제 생각엔 작동이 안되는 코드를 빼고 다시 커밋한 다음, 스샷 찍는 PR을 날리는게 좋지 않을까 싶습니다.
- PR 날리는 원칙을 같이 정해볼 필요가 있을 것 같아요. 제가 느끼기에는 크기가 굉장히 큰 PR인 것 같은데, 확인하기가 어렵다는 느낌을 받았어요. 같이 얘기해봐요!
- 어떤 형식의 파일을 올려도 되는지를 확실히 해야 할 것 같아요. 저도 Unity를 잘 몰라서 막 올렸는데, ignore 질을 같이 해봅시다 ㅎㅎ

나머지 내용은 좋구요, refactoring은 또 같이 해보면 될 것 같네요. 고생 많이 했어요!

Cliwo committed:
Change ScreenCapture Fallback feature
-> Now Unity captures screen automatically based on 'autoCaptureInterval' of ScreenCaptureManager.cs
